Keas.Mvc/Controllers/Api/KeySerialsController.cs

```diff
@@ -138,18 +152,18 @@ public async Task<IActionResult> Create([FromBody] CreateKeySerialViewModel mode
             // create key serial
             var keySerial = new KeySerial
             {
-                KeyId    = key.Id,
-                Key      = key,
-                TeamId   = key.TeamId,
-                Name     = model.Number.Trim(),
-                Number   = model.Number.Trim(),
-                Notes    = model.Notes, 
-                Status   = model.Status ?? "Active",                
+                KeyId = key.Id,
+                Key = key,
+                TeamId = key.TeamId,
+                Name = model.Number.Trim(),
+                Number = model.Number.Trim(),
+                Notes = model.Notes,
+                Status = model.Status ?? "Active",
```

Comment:
haha John is barely gone and then you do this to him!

Reply:
Status should not be null (I fixed that)